### ISSUE TITLE
Error while loading budget

### ISSUE THREAD

**Issue by efess** [duplaja/actual-simplefin-sync#13]
I'm trying to load the app for the first time, and after running through setup, I'm getting an error: `Error updating Error: out-of-sync-migrations` which has `TypeError: Cannot read properties of undefined (reading 'timestamp')` further up the error stack.

Looks like this is caused by the fact `@actual-app` is performing migrations on a local copy of the budget db and it's calling `getClock` but the clock has not been initialized yet.

**Comment by efess** [comment]
Apparently this is an issue with actual itself, not this project - so will open up an issue there.

**Comment by jacobkossman** [comment]
> Apparently this is an issue with actual itself, not this project - so will open up an issue there.

did you ever open an issue or get this fixed? running into this myself.

**Comment by efess** [comment]
@jacobkossman  Yes, this project had an outdated reference to the actual api. Updating that fixed it.

**Comment by jacobkossman** [comment]
@efess i was updated to the latest version, but still getting that error. did you have to manually update it? can you share that? thanks
